authentication/views.py

```diff
@@ -60,12 +60,14 @@ def forgot_view(request):
 
 def forgot_password(request):
     if request.method == 'POST':
-        try:
-            user = User.objects.get(email=request.POST['email'].lower())
-            user.profile.send_password_email(get_current_site(request).name)
-            return JsonResponse(success_response)
-        except User.DoesNotExist:
-            return JsonResponse(failure_response)
+        if not request.POST['email'] == '': 
```

Comment:
you don't need the == '' you can drop that and just have it if not request.POST['email']
python implicitly handles nulls and empty this way. By doing == ' ' you're no longer doing the null check.